### ISSUE TITLE
Add .eml as supported file

### ISSUE THREAD

**Issue by BastianZim** [gotenberg/gotenberg#565]
Hi, this is more a general feature request and maybe something that should be opened in the PDF engine repos so please feel free to close if inappropriate.

I was wondering if it would be possible to add emails (.eml) as a supported file format. There is just no tool that can convert emails currently, so would be really helpful.

Thanks :)

**Comment by ra-coder** [comment]
in eml could be attached files in different formats 
So I in my project parse eml in python, and send separately body of email and attached files to gotenberg

**Comment by BastianZim** [comment]
Yes, that could work. I'm more thinking of converting the eml, including recipient etc., to a pdf. Similar to what mhonarc is doing.

**Comment by gulien** [comment]
@BastianZim Do you know if it works with LibreOffice? Gotenberg is mostly a wrapper around powertools like LibreOffice, so if one of these tools can do the job, it could be added quite easily.

**Comment by BastianZim** [comment]
Unfortunately not. To be honest, I have searched high and low and haven't found anything anywhere besides mhonarc that is able to do so. So this is probably not something that can be implemented short term.

**Comment by gulien** [comment]
I'm closing this issue. Please re-open it if you find a viable implementation.
